canteen/templates/canteen/login.html

```diff
@@ -16,6 +16,7 @@
             <input class="Grake-button" type="submit" value="登录">
         </div>
     </form>
-    <p style="width: 250px; text-align: left; font-size: 0.6em; color: #455A64;">本服务由学生提供，并非深圳实验学校高中部官方作品。我们不对使用本网站造成的一切损失，包括但不限于：订餐提交失败、取餐时发现显示屏上只有一份青菜等情况负责。若您不能接受，请关闭本网页并于浏览器访问深圳实验学校官方订餐系统 <a href="http://gzb.szsy.cn/card/">http://gzb.szsy.cn/card/</a></p>
+    <p style="width: 250px; text-align: left; font-size: 0.6em; color: #455A64;">不妨尝试<a href="https://github.com/szes-maker/EAxT/releases/download/v0.2.1/org.szesmaker.ordermeal_v0.2.1.apk" target="_blank">食验客户端</a></p>
```

Comment:
I will do some style changes after merging it.

Reply:
Shall we add a simple Javascript to detect the user's browser? This kind of promotion is pointless on Apple devices.
Maybe just show it on Android and Windows?